### ISSUE TITLE
Define Accessibility Artifacts per Project Phase / Step

### ISSUE THREAD

**Issue by bischoffa** [department-of-veterans-affairs/va-mobile-app#9088]
**Description:** 

**Notes:** 

**AC**

**Comment by brea11y** [comment]
I have created a [Google doc here](https://docs.google.com/document/d/1VaZuurcgR5Bk3OpIWGT8bRyWoCzgy-P0grqAtDJQbsA/edit?usp=sharing) where I have imported the steps from the Google slides presentation. In each step, I have highlighted areas in yellow where accessibility should be included and aqua/blue where an artifact will be required.

**Comment by bischoffa** [comment]
Reviewed and Brea and approve of the milestones. 

Few things to think about: 

- Creating a template for annotated work 
- Thinking about how to handle / process for when annotated work is completed but the design changes later 
- Should Accessibility have a section in the QA Signoff doc (first used in HSP)

**Comment by brea11y** [comment]
Will review this with @TKDickson during our 1:1. Will open new tickets based on that conversation.
